src/AuthenticationPortal.Contracts/Contracts/ITokenAuthenticator.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System.Threading.Tasks;
+
+namespace AuthenticationPortal.Contracts
+{
+    public interface ITokenAuthenticator
+    {
+        Task<IActionResult> AuthenticateToken(Token token);
```

Comment:
prefix with async

Reply:
I cannot prefix async to it. Its giving me error. It states the 'async' modifier can only be used in methods that have a body

Reply:
prefix the method name with async `AuthenticateTokenAsync` (whenever it is async operation for readability)

Reply:
> prefix the method name with async `AuthenticateTokenAsync` (whenever it is async operation for readability)

Oh, Okay

Reply:
Done